Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -165,7 +165,8 @@ @interface ConversationViewController () <AVAudioPlayerDelegate,
 @property (nonatomic, nullable) OWSAudioAttachmentPlayer *audioAttachmentPlayer;
 @property (nonatomic, nullable) NSUUID *voiceMessageUUID;
 
-@property (nonatomic) NSTimer *readTimer;
+@property (nonatomic, nullable) NSTimer *readTimer;
+@property (nonatomic, nullable) NSTimer *cellMediaCachesTimer;
```

Comment:
We clear cached cell media using a "load window" around the visible cells, which is expensive enough that we want to debounce & throttle that work, hence the timer.